en/docs/get-started/tutorials/create-your-first-event-triggered-integration.md

```diff
@@ -68,27 +68,29 @@ Let's create the integration component by following the steps given below:
 
 
 ## Step 2: Deploy
+
 Before you deploy the component, you must follow the steps given below to assign appropriate RabbitMQ credentials to the environment variables:
 
-1. In the **Deploy** view, go to the **Development** card and click **Configure via the DevOps Portal**. This takes you to the **Configs & Secrets** page in the **DevOps** profile, where you can apply environment-specific configuration files, environment variables, and other secret resources to containers.
-2. Click **Let's Get Started**.
-3. In the **Mount a Configuration** page, select **Environment Variables** as the configuration type and click **Next**.
-4. Select **Create New** and enter a **Config Name**. Here, let's specify `environmentconf` as the **Config Name**.
-5. Click **Next**.
-6. Specify the environment variables as key-value pairs. You must use the values you obtain from your RabbitMQ instance as the values for the following keys: 
+1. In the **Deploy** view, go to the **Build Area** card, and then click **Configure & Deploy**.
+2. In the **Configure & Deploy** pane, add the environment variables as key-value pairs. You must use the values you obtain from your RabbitMQ instance as the values for the following keys: 
 
     | **Key**    |
     |------------|
     | HOSTNAME   | 
     | USERNAME   |
     | PASSWORD   | 
     | VHOST      | 
-
-7. Click **Finish**. Now you are ready to deploy the component to the developer environment. 
-
-8. Go to the **iPaaS** profile and click the `MQTT listener` component you created.
-
-9. Click **Deploy** on the left navigation, go to the **Build Area** card, and then click **Deploy Manually**.
+3. Click **Deploy**. 
```

Comment:
Might have to try out and see whether the change in the order of steps is correct. From what I recall it was necessary to do the steps in the DevOps profile before clicking Deploy.

Reply:
Verified with the team, the order is fine as the above is given as an alternative path.